migrations/20210520035423-add-program-director-references.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
```

Comment:
Here I create the foreign key relationships, as I had demonstrated before between User and Student... now between ProgramDirector and Organization and User